### DISCUSSION TITLE
Вернуть возможность паунса в пулеметчика

### DISCUSSION THREAD

**Discussion by Shootnik01** [Cosmic-Overlord/TerraGov-Marine-Corps#332] [Ideas]
Ибо ну это фигня какая-то. Его даже прыжком в спину не то что снять с пулемета, а даже продамажить не выйдет.

**Comment by fe-ed** [comment]
Можно просто сделать тайл с пулеметом проходимым для всех

**Reply by Shootnik01** [reply]
Ну хз, мне  кажется, можно будет смешно абузить. Почему как ревертнуть его изменения нельзя, чтобы чел считался стоящим за пулеметом, а не в отдельном измерении закрепленным внутри? Можно сделать так, чтобы его толкать не могли, раз мары плачут, на выкидывание из него
